connector/elasticapmconnector/README.md

```diff
@@ -44,3 +44,19 @@ elasticapm:
     directory: /path/to/aggregation/directory
     metadata_keys: [list, of, metadata, keys]
 ```
+
+### Metrics produced by the connector
+
+| Metric                                          | Source Signal       | OTel Metric Type     | ES Mapping Type         |
+| ----------------------------------------------- | ------------------- | -------------------- | ----------------------- |
+| `service_summary`                               | Logs, Metric, Spans | Sum                  | Double                  |
+| `span.destination.service.response_time.sum.us` | Spans               | Sum                  | Double                  |
+| `span.destination.service.response_time.count`  | Spans               | Sum                  | Double                  |
+| `transaction.duration.histogram`                | Spans               | Histogram            | [Histogram](https://www.elastic.co/guide/en/elasticsearch/reference/current/histogram.html)               |
+| `transaction.duration.summary`                  | Spans               | Histogram (1 bucket) | [Aggregate Metric Double](https://www.elastic.co/guide/en/elasticsearch/reference/current/aggregate-metric-double.html) |
+| `event.success_count`                           | Spans               | Histogram (1 bucket) | [Aggregate Metric Double](https://www.elastic.co/guide/en/elasticsearch/reference/current/aggregate-metric-double.html) |
```

Comment:
nit: Can we add the success_count is based on event.outcome?

Reply:
Done!